3_predictions.yml

```diff
@@ -5,21 +5,97 @@ packages:
   - dplyr
   - feather
   - lubridate
+  - yaml
 
 sources:
   - 3_predictions/src/munge_predictions.R
+  - 3_predictions/src/get_prms_sntemp_model.R
+  - 3_predictions/src/set_sntemp_output.R
+  - 3_predictions/src/update_sntemp.R
+  - 3_predictions/src/run_sntemp.R
+  - 3_predictions/src/get_sntemp_values.R
+  - 3_predictions/src/gather_sntemp_output.R
+  - 3_predictions/src/get_subbasins.R
+  - 3_predictions/src/get_upstream_downstream_segs.R
 
 targets:
   3_predictions:
     depends:
      - 3_predictions/out/compare_predictions_obs.feather.ind
      - 3_predictions/out/model_evaluation.csv
 
-  # pull out predictions from SNTEMP in/out - updated in April 2020
-  # to use drivers that run through 2019
+  # getting some settings for running PRMS-SNTemp uncalibrated
+  uncal_settings:
+    command: read_yaml('3_predictions/cfg/uncal_settings.yml')
+
+  orig_model_loc:
+    command: uncal_settings[[I('orig_model_loc')]]
+
+  uncal_model_run_loc:
+    command: uncal_settings[[I('uncal_model_run_loc')]]
+
+  uncal_start:
+    command: uncal_settings[[I('start')]]
+
+  uncal_stop:
+    command: uncal_settings[[I('stop')]]
+
+  sntemp_output_vars:
+    command: uncal_settings[[I('sntemp_output')]]
+
+  # need to make sure that 20191002_Delaware_streamtemp folder is in project on
+  #  local machine; this is on GD if not on local machine
+  # within this function, moving model to temporary run location so that we can change parameters, etc.. while keeping
+  #  original model preserved
+  get_prms_sntemp:
+    command: get_prms_sntemp_model(
+      gd_zip_ind_file = '3_predictions/in/prms_sntemp/20191002_Delaware_streamtemp.zip.ind',
+      unzip_loc = I('prms_sntemp'),
+      orig_model_loc = orig_model_loc,
+      model_run_loc = uncal_model_run_loc)
+
+  set_sntemp_output_vars:
+    command: set_sntemp_output(
+      output_names = sntemp_output_vars,
+      model_run_loc = uncal_model_run_loc)
+    depends: get_prms_sntemp
+
+  uncal_sntemp_run:
+    command: run_sntemp(
+      start = uncal_start,
+      stop = uncal_stop,
+      spinup = I('F'),
+      restart = I('F'),
+      model_run_loc = uncal_model_run_loc)
+    depends: set_sntemp_output_vars
+
```

Comment:
Same thing for the uncal run. Because it is a target, everyone will need to run. I would write a file and indicate so we can git commit the indicator and skip rerunning this model each time someone is new to the project.